infrastructure/stacks/staging_pipeline_stack.py

```diff
@@ -0,0 +1,116 @@
+from aws_cdk import (
+    RemovalPolicy,
+    Stack,
+)
+from aws_cdk import (
+    aws_codepipeline as codepipeline,
+)
+from aws_cdk import (
+    aws_codepipeline_actions as cpactions,
+)
+from aws_cdk import (
+    aws_iam as iam,
+)
+from aws_cdk import (
+    aws_s3 as s3,
+)
+from aws_cdk import (
+    aws_ssm as ssm,
+    aws_codebuild as codebuild,
+)
+from constructs import Construct
+
+from stacks.staging_stack import STAGING_APP_NAME, STAGING_ENV_NAME
+
+
+class StagingPipelineStack(Stack):
+    def __init__(
+        self,
+        scope: Construct,
+        construct_id: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        artifact_bucket = s3.Bucket(
+            self,
+            "StagingArtifactBucket",
+            removal_policy=RemovalPolicy.DESTROY,
+            auto_delete_objects=True,
+        )
+        source_output = codepipeline.Artifact()
+        build_output = codepipeline.Artifact()
```

Comment:
we don't use the build stage in our current pipelines so we can probably just remove the bits in this file related to that